app/routes/index.js

```diff
@@ -0,0 +1,17 @@
+const choo = require('choo');
+const download = require('./download');
+
+const app = choo();
+
+app.route('/', require('./home'));
+app.route('/share/:id', require('../templates/share'));
+app.route('/download/:id', download);
+app.route('/download/:id/:key', download);
+app.route('/completed', require('../templates/completed'));
+app.route('/unsupported/:reason', require('../templates/unsupported'));
+app.route('/legal', require('../templates/legal'));
+app.route('/error', require('../templates/error'));
+app.route('/blank', require('../templates/blank'));
+app.route('*', require('../templates/notFound'));
```

Comment:
I haven't tested this locally yet, but... don't we need the `/__version__` route, as well as the two health check OPs routes?

Reply:
These are routes relative to the frontend js app. Server-only routes are in `server/routes/index.js`